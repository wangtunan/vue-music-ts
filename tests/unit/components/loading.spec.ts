import { shallowMount, Wrapper } from '@vue/test-utils'
import Loading from '@/components/loading/index.vue'

describe('loading.vue', () => {
  let wrapper: Wrapper<Vue>
  const title = '正在加载中...'
  beforeEach(() => {
    wrapper = shallowMount(Loading, {
      propsData: {
        title: title
      }
    })
  })

  it('test snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
  it('test props.title', () => {
    expect(wrapper.text()).toContain(title)
  })
})