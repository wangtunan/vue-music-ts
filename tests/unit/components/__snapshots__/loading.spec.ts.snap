// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading.vue test snapshot 1`] = `
<div class="m-loading">
  <div class="loading-content"><img src="@/assets/images/loading.gif" alt="">
    <p class="title">正在加载中...</p>
  </div>
</div>
`;
